kubetest/extract.go

```diff
@@ -68,16 +68,16 @@ func (l *extractStrategies) String() string {
 // Converts --extract=release/stable, etc into an extractStrategy{}
 func (l *extractStrategies) Set(value string) error {
 	var strategies = map[string]extractMode{
-		`^(local)`:                  local,
-		`^gke-?(staging|test)?$`:    gke,
-		`^gci/([\w-]+)$`:            gci,
-		`^gci/([\w-]+)/(.+)$`:       gciCi,
-		`^ci/(.+)$`:                 ci,
-		`^release/(latest.*)$`:      rc,
-		`^release/(stable.*)$`:      stable,
-		`^(v\d+\.\d+\.\d+[\w.-]*)$`: version,
-		`^(gs://.*)$`:               gcs,
-		`^(bazel/.*)$`:              bazel,
+		`^(local)`:                    local,
+		`^gke-?(staging|test)?$`:      gke,
+		`^gci/([\w-]+)$`:              gci,
+		`^gci/([\w-]+)/(.+)$`:         gciCi,
+		`^ci/(.+)$`:                   ci,
+		`^release/(latest.*)$`:        rc,
+		`^release/(stable.*)$`:        stable,
+		`^(v\d+\.\d+\.\d+[\w.\-+]*)$`: version,
```

Comment:
(you don't need to change this, but a tip: if you put `-` at the start of the character class, you don't need to escape it, i.e.
 something like `[-\w.+]` would work.)

Reply:
I see, good to know, thanks!